frontend/src/components/Layouts/HomePageComponent/HomePageComponent.jsx

```diff
@@ -1,13 +1,54 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import CategoryOutput from './components/CategoryOutput';
 import Carousel from './components/Carousel';
 import carouselData from '../../../data/carouselData.json';
 import styles from './HomePageComponent.module.scss';
+// eslint-disable-next-line max-len
+import { GetCategories } from '../../../services/catalog/categoryAccess.service';
+import { useWindowSize } from '../../../hooks/useWindowSize';
+import SmallCategories from './components/SmallCategories/SmallCategories';
 
 const HomePageComponent = () => {
+  const { width } = useWindowSize();
+  const [mainCategories, setMainCategories] = useState(null);
+
+  useEffect(() => {
+    if (width < 868) {
+      const getCategories = async () => {
+        try {
+          const categories = await GetCategories();
+          const mainCategories = categories.filter(
+            (category) => category.parent === null,
+          );
+          setMainCategories(mainCategories);
+        } catch (error) {
+          console.error(error);
+        }
+      };
+      getCategories();
+    }
+  }, [width]);
   return (
     <>
       <Carousel data={carouselData.slides} />
+      {mainCategories && (
+        <ul className={styles.categorySlider}>
+          {mainCategories.map((category) => {
+            return (
+              <li key={category.alias}>
```

Comment:
Why do you `category.alias` use as a key ? Why not `category.id`?

Reply:
Because it's more specific